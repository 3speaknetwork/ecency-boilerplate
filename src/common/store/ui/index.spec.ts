import reducer, {initialState, toggleLoginAct, toggleSignUpAct} from "./index";

let state = initialState;

it("1- default state", () => {
    expect(state).toMatchSnapshot();
});

it("2- toggleLoginAct", () => {
    state = reducer(state, toggleLoginAct());
    expect(state).toMatchSnapshot();
});

it("3- toggleSignUpAct", () => {
    state = reducer(state, toggleSignUpAct());
    expect(state).toMatchSnapshot();
});

it("4- toggleLoginAct", () => {
    state = reducer(state, toggleLoginAct());
    expect(state).toMatchSnapshot();
});
