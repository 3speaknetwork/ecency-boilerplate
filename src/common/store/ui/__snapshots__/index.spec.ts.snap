// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1- default state 1`] = `
Object {
  "login": false,
  "signUp": false,
}
`;

exports[`2- toggleLoginAct 1`] = `
Object {
  "login": true,
  "signUp": false,
}
`;

exports[`3- toggleSignUpAct 1`] = `
Object {
  "login": true,
  "signUp": true,
}
`;

exports[`4- toggleLoginAct 1`] = `
Object {
  "login": false,
  "signUp": true,
}
`;
